### DISCUSSION TITLE
Welcome to horus-heresy Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [BSData/horus-heresy#1871] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mayegelt** [comment]
I'm Maye. I started doing edits after playing around and learning the ropes on an partially abandoned repo called Gates of Antares when it was still kind of new. Migrated over to do Heresy fixes, and tend to start doing small fixes, but the longer they get left, the more fixes get added to my pull requests.
I tend to like to focus on fixing stuff to unwind, while also watching TV
